src/richie/apps/demo/management/commands/create_demo_site.py

```diff
@@ -1,19 +1,17 @@
 """Create_demo_site management command."""
+from collections import defaultdict
 import logging
 import random
-from collections import defaultdict
 
+from cms.api import add_plugin
+from cms.models import StaticPlaceholder
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.core.management.base import BaseCommand, CommandError
 from django.test.utils import override_settings
 from django.utils import translation
 from django.utils.translation import gettext_lazy as _
-
 import factory
-from cms.api import add_plugin
-from cms.models import StaticPlaceholder
```

Comment:
I'm wondering why cms moved to the django block :thinking: 
Normally we wanted to have a block just for django as defined here: https://github.com/openfun/richie/blob/master/setup.cfg#L111

Reply:
Just my IDE that sorts imports on save but it is not yet compliant with isort rule so lint-back fails 😅

Reply:
Oh ok! I was anticipating some potholing in isort. Better like this :sweat_smile: